### PR TITLE
isReply helper function

### PR DESCRIPTION
**Pull request by nikkilr88**
I am still pretty new at this...
Is this what you were looking for?

### CODE REVIEW

File: src/isReply.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const isReply = (tweet) => {
+    var RT = /^RT/i;
+    
+    if (RT.test(tweet.text) ||
```
Comment (spences10) [inline src/isReply.js]: I'm guessing you have tested `.test(` on your own bot successfully?

File: src/isReply.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        tweet.is_quote_status ||
+        tweet.retweeted_status ||
+        tweet.in_reply_to_status_id ||
+        tweet.in_reply_to_status_id_str ||
+        tweet.in_reply_to_user_id ||
+        tweet.in_reply_to_user_id_str ||
+        tweet.in_reply_to_screen_name)
+        return true;
```
Comment (spences10) [inline src/isReply.js]: Formatting isn't in line with rest of project, I can clean this up if you like 

I have a PR I'm waiting on some changes for but the Prettier formatting rule I'd prefer for here is:

```json
"prettier es5 --no-semi --single-quote --write src/**/*.js"
```

In your change it's only semicolons need removing

File: src/isReply.js (hunk 1 above)
Comment (nikkilr88) [reply src/isReply.js]: Yes... It filters out tweets that begin with "RT".

Comment (nikkilr88) [reply src/isReply.js]: Should I take that out?

**Comment by nikkilr88** [comment]
https://github.com/spences10/twitter-bot-bootstrap/blob/fc307eea5faac36c00a4e96617bd64df37332b0f/src/api/retweet.js#L31


Would that be an if statement inside of your try block then?

**Comment by spences10** [comment]
I think if we get rid of the try and use the `isReply` function in an `if ` it should do the trick, you think?

**Comment by spences10** [comment]
Closes #120

**Comment by spences10** [comment]
Ok, looks great, you tested this on your own bot I presume @nikkilr88? 🙏 👍

**Comment by nikkilr88** [comment]
@spences10 Yes, my bot uses the isReply function, and so far everything is running smoothly. It filters out retweets and quoted tweets.

**Comment by spences10** [comment]
Brilliant, thank you ❤️

**Comment by nikkilr88** [comment]
@spences10 Thank you! This was my first PR that actually involved code. It was a great learning experience.

**Comment by spences10** [comment]
Thank you for the contribution 👍 

I'll get the contributor's file updated now, please feel free to make any suggestions + improvements you think may help other people trying this out.

I have found that this is a nice entry level project for people to try

Many thanks 👌

**Pull request merged**